### PR TITLE
Fixed lowMach threshold for velocity freestream

### PR DESCRIPTION
**Pull request by NasserFlexCompute**

### CODE REVIEW

File: flow360/component/flow360_params/solvers.py [hunk 1]
```diff
@@ -204,7 +204,10 @@ def to_solver(self, params, **kwargs) -> NavierStokesSolver:
 
         if self.low_mach_preconditioner:
             if self.low_mach_preconditioner_threshold is None:
-                self.low_mach_preconditioner_threshold = params.freestream.Mach
+                if hasattr(params.freestream, "Mach"):
+                    self.low_mach_preconditioner_threshold = params.freestream.Mach
+                else:
+                    self.low_mach_preconditioner_threshold = params.freestream.getMach()
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/solvers.py:210]: I think enough would be to just do:
```suggestion
                self.low_mach_preconditioner_threshold = params.freestream.to_solver(params, **kwargs).Mach                    
```
and other changes are not required. The `to_solver()` returns object that has Mach calculated

File: flow360/component/flow360_params/flow360_params.py
Comment (benflexcompute) [inline flow360/component/flow360_params/flow360_params.py:1]: Can we add a unit test to ensure that the low_mach_preconditioner_threshold is properly set depending on the freestream condition?

File: flow360/component/flow360_params/solvers.py (hunk 1 above)
Comment (maciej-flexcompute) [reply flow360/component/flow360_params/solvers.py:210]: but the issue is that Mach does not exist in all types of Freestream. Only after to_solver() we have access to Mach.

Comment (benflexcompute) [reply flow360/component/flow360_params/solvers.py:210]: right we need the value. Ignore my comment then.

**Pull request closed without merging**